src/modules/pv/pv_core.c

```diff
@@ -3238,11 +3238,11 @@ int pv_get_msg_attrs(sip_msg_t *msg, pv_param_t *param, pv_value_t *res)
 
 int pv_parse_env_name(pv_spec_p sp, str *in)
 {
+	char *csname;
+
 	if(in->s==NULL || in->len<=0)
```

Comment:
it does not look like any of the parameters can be null, judging by the checks done before calling this function in https://github.com/kamailio/kamailio/blob/master/src/core/pvapi.c#L873, but I may be missing something else